packages/kernel/src/index.ts

```diff
@@ -1,5 +1,4 @@
 export * from "./kernel";
 export * from "./streamlit-replacements/lib/ConnectionManager";
-export * from "./streamlit-replacements/lib/FileUploadClient";
```

Comment:
Memo: The patched `FileUploadClient` has been moved to `whitphx/streamlit` in https://github.com/whitphx/streamlit/pull/7